app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose.home
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.sp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.SectionHeader
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Homepage header.
+ *
+ * @param text The header string.
+ * @param (optional) onShowAllClick Action to take when show all is clicked.
+ */
+@Composable
+fun HomeSectionHeader(
+    text: String,
+    description: String,
```

Comment:
```suggestion
    contentDescription: String,
```
Can you also add an @param

Reply:
Were we unable to call this `contentDescription`?